tests/foreman/cli/test_remoteexecution.py

```diff
@@ -255,7 +255,7 @@ def test_positive_run_custom_job_template_by_ip(self, rex_contenthost, module_or
 
     @pytest.mark.destructive
     @pytest.mark.rhel_ver_list([7])
```

Comment:
You can do `@pytest.mark.rhel_ver_list([7,8,9])`  as well right ?? Is there any reason why it's not done that way?

Reply:
I do it where it makes sense but I didn't want to quadruple the whole module for little gain. Here we test mostly a server-side procedure that is unlikely to be influenced by the target host OS.